### PR TITLE
Change tolerance

### PR DESCRIPTION
**Pull request by turbolay**
See https://github.com/zkSNACKs/WalletWasabi/pull/10741
Parameters might be changed, I didn't research which would be best.
I tried to make them large enough to avoid too much determinism in the decompositions.

Method could also be extracted.

### CODE REVIEW

File: Sake/Mixer.cs [hunk 1]
```diff
@@ -335,15 +335,26 @@ public IEnumerable<Output> Decompose(IEnumerable<Money> myInputsParam, IEnumerab
             preCandidates.Shuffle();
 
             var orderedCandidates = preCandidates
-                .OrderBy(x => x.Decomp.All(x => denomHashSet.Contains(x)) ? 0 : 1) // Prefer no change.
+                .OrderBy(x => x.Decomp.Sum(y => denomHashSet.Contains(y) ? 0 : y.Amount)) // Prefer lower change.
```
Comment (adamPetho) [inline Sake/Mixer.cs]: Can't/shouldn't we use the `CalculateChange` method? This PR introduces that method.

Or is here something I don't see?

Comment (adamPetho) [inline Sake/Mixer.cs]: General question:

Why do we shuffle the `preCandidates` if we order it right away?

**Comment by turbolay** [comment]
I think that https://github.com/zkSNACKs/WalletWasabi/pull/10777 is an alternate and made this PR useless.

**Comment by turbolay** [comment]
I'm reopening because after discussion with @adamPetho it seems that it might still be relevant, even if less important.
I let you @nopara73 the choice to close it or request changes for implementation.

**Comment by nopara73** [comment]
https://github.com/nopara73/Sake/pull/32

**Pull request closed without merging**